src/util/Util.js

```diff
@@ -81,7 +81,7 @@ class Util {
   static parseEmoji(text) {
     if (text.includes('%')) text = decodeURIComponent(text);
     if (text.includes(':')) {
-      const [name, id] = text.split(':');
+      const [, name, id] = text.match(/<:([\w_]{2,32}):(\d+)>/);
```

Comment:
This errors out when using `Message#react` with a custom emoji, since that will pass the emoji as `name:id` which does not match the regex.
You could change the regex to make the `<:` and `>` optional to fix this, something like `/(?:<:)?([\w_]{2,32}):(\d+)>?/` should do it.